### ISSUE TITLE
R session Aborted when calling spacy_initialize

### ISSUE THREAD

**Issue by ttfg22** [quanteda/spacyr#235]
I am working on Windows 10 and used spacyr without problems. At a certain point spacy R always caused the already metioned 'R session aborted'. I unistalled and tried to reinstall spacyR with spacy_install and also in the anaconda promp, but the installtion didn't end. I could reinstall spacy via miniforge, but when I tried to connect from R (spacy_initialize) there is still the 'R Session abortion' after 
' Found 'spacy_condaenv'. Spacy will use this environment'.  
What else can I do in order to use this wonderful package again?
Thank you for any help

Session Info R: 
```
R version 4.1.3 (2022-03-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.1252  LC_CTYPE=German_Germany.1252    LC_MONETARY=German_Germany.1252 LC_NUMERIC=C                   
[5] LC_TIME=German_Germany.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.9      R6_2.5.1        lifecycle_1.0.3 DBI_1.1.3       httr_1.4.4      rlang_1.0.6     cli_3.4.1       rstudioapi_0.14
 [9] blob_1.2.3      vctrs_0.4.2     ellipsis_0.3.2  RPostgres_1.4.4 tools_4.1.3     bit64_4.0.5     bit_4.0.4       hms_1.1.2      
[17] compiler_4.1.3  pkgconfig_2.0.3
```

**Comment by DavidLenS** [comment]
I ran into the same problem. Ultimately, I tried if switching the version of R would work. I switched the version of R to use from 4.1.2 to 4.0.0 (Tools -> Global Options -> General -> Basic -> R version: -> Change), and it worked. I hope this helps.

**Comment by ElisaWirsching** [comment]
I am having the same issue and unfortunately @DavidLenS's solution didn't work for me. 

@ttfg22, did you manage to solve this problem on Windows 10?

**Comment by ttfg22** [comment]
I could solve the problem by activating the spacy environment via the miniforge command line. When I called spacy_initialize then, everything worked out. You could maybe try to install a newer version of R? I am using R version 4.2.2.

**Comment by kbenoit** [comment]
Solved in v1.3.
